install/rhel-installation-guide.md

```diff
@@ -1,19 +1,11 @@
 # Install Kata Containers on RHEL
 
-> **Warning:**
->
-> - The RHEL packages are provided as a convenience to users until native
->   packages are available in RHEL. However, they are **NOT** currently tested
->   (although CentOS is) so caution should be exercised.
->
->   See https://github.com/kata-containers/ci/issues/3 for further details.
-
 1. Install the Kata Containers components with the following commands:
 
    ```bash
    $ source /etc/os-release
    $ ARCH=$(arch)
-   $ sudo -E yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/RHEL_${VERSION_ID}/home:katacontainers:releases:${ARCH}:master.repo"
+   $ sudo -E yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/RHEL_7/home:katacontainers:releases:${ARCH}:master.repo"
```

Comment:
Why is this change required? We really don't want to hard-code the `7` in there (RHEL 8 is in beta...).

Reply:
@jodh-intel that is the OBS url, ID is equal to "rhel" without including a number or version

Reply:
@jodh-intel version id does not match the obs url

Reply:
cat /etc/os-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.6 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.6"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.6 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.6:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

Reply:
In which case, I'd do something like:

```sh
$ VERSION=$(echo $VERSION_ID|cut -d\. -f1)
```

Then, use `RHEL_${VERSION}` in the URL.